src/core/api/clock.ts

```diff
@@ -98,7 +99,17 @@ export interface IStalledStatus {
 export interface IClockTick extends IMediaInfos {
   /** Set if the player is stalled, `null` if not. */
   stalled : IStalledStatus | null;
-  getCurrentTime : () => number;
+}
+
+export interface IClock {
```

Comment:
JSDoc (`/**`) might be useful here.
Also isn't the name too generic? It sounds to me like it miss a word, don't know what though :p